vendor/sigs.k8s.io/controller-runtime/pkg/client/fake/client.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
Is this a carry patch on top of vendored `controller-runtime`? If so, please use `UPSTREAM: <carry>: openshift:` prefix in the commit header.

Reply:
@ingvagabund Can you please clarify this comment?

Reply:
`s/Hack vendor to make label selector work under the fake client/UPSTREAM: <carry>: openshift: UPSTREAM: <carry>: openshift`

Reply:
@ingvagabund Do you need to set `UPSTREAM: <carry>: openshift:` for each commit that does changes under the vendor directory?

Reply:
Any commit where you change the code without running `dep ensure` would be helpful. So we can backtrace who changed the code under vendor and possibly why.

Reply:
done